lib/3scale_toolbox/remotes.rb

```diff
@@ -0,0 +1,68 @@
+module ThreeScaleToolbox
+  module Remotes
+    def self.included(base)
+      base.extend(ClassMethods)
+    end
+    module ClassMethods
+    end
+
+    ##
+    # Fetch remote list
+    # Perform validation
+    #
+    def remotes
+      rmts = config.data :remotes
+      return invalid_remote unless validate_remotes?(rmts)
```

Comment:
I think this would be better as `raise invalid_remote` as this `return` implies it should return something, but in fact it never does.